src/source/reporting-query-and-response.rst

```diff
@@ -0,0 +1,94 @@
+
+Query
+************************************
+The metrics query should be provided as a JSON object of type `Metrics Request`_, which has these minimum requirements:
+
+* At least one valid entry in the  :ref:`Time Ranges <time-ranges>` field.
+
+* At least one valid entry in the :ref:`Metrics <metrics>` field.
+
+Here is a sample request with all top-level query fields expanded:
+
+.. code-block:: javascript
+
+    POST /api/sellside/metrics/data
+    Content-Type: application/sellside.metrics.data-v1+json
+    Accept: application/json
+.. literalinclude:: /examples/metrics-query-v1.json
+
+Metrics Request
+######################################################
+
+The data model of the request has the following structure:
+
+.. code-block:: javascript
+
+    {
+        "timeRanges": [<timeRange>, ...],
+        "aggregate": <string>,
+        "dimensions": [<string>, ...],
+        "metrics": [<string>, ...],
+        "sorts": [<sort>, ...],
+        "filters": [<filter>, ...],
+        "enrichment": [<string>, ...],
+        "limit": <integer>,
+        "offset": <integer>,
+        "searchPhrase": <string>
+    }
+
+
+================  ======================================
+Parameter          Description
+================  ======================================
+``timeRanges``     Array of :ref:`Time Ranges <time-ranges>` objects, separated by commas
+``aggregate``      One of :ref:`Time Aggregation <time-aggregation>` string values
+``dimensions``     Array of dimensions, separated by commas
+``metrics``        Array metrics, separated by commas. At least one element is mandatory
+``sorts``          Array :ref:`Sort <Sorts>`  objects, separated by commas
+``filters``        Array :ref:`Filters <filters>` objects, separated by commas
+``limit``          Number of elements in the response, per time range. By default there is no limit
```

Comment:
don't we have an internal limit? if not, do we want to add it?

Reply:
Maybe...It's hard to say what the right limit is, ... even if there are a lot of rows, maybe each row is lightweight in terms of data